src/LmcCookieConsentManager.js

```diff
@@ -26,11 +27,15 @@ const defaultOptions = {
  * @param {string} [args.defaultLang] - Default language. Must be one of predefined languages.
  * @param {boolean} [args.autodetectLang] - Autodetect language from the browser
  * @param {function} [args.onFirstAccept] - Callback to be executed right after any consent is just accepted
- * @param {function} [args.onFirstAcceptOnlyNecessary] - Callback to be executed right after only necessary cookies are accepted
+ * @param {function} [args.onFirstAcceptOnlyNecessary] - Callback to be executed right after only necessary cookies
+ *                                                       are accepted
  * @param {function} [args.onFirstAcceptAll] - Callback to be executed right after all cookies are accepted
- * @param {function} [args.onAccept] - Callback to be executed when any consent is detected (either given right now or already saved previously)
- * @param {function} [args.onAcceptOnlyNecessary] - Callback to be executed when consent with only necessary cookies is detected (either given right now or already saved previously)
- * @param {function} [args.onAcceptAll] - Callback to be executed when consent with all cookies is detected (either given right now or already saved previously)
+ * @param {function} [args.onAccept] - Callback to be executed when any consent is detected (either given right now
+ *                                     or already saved previously)
+ * @param {function} [args.onAcceptOnlyNecessary] - Callback to be executed when consent with only necessary cookies
+ *                                                  is detected (either given right now or already saved previously)
```

Comment:
Just a personal preference, but this seems ugly to me :smile:  .

Cannot it break it like at least like this?

```suggestion
 * @param {function} [args.onAcceptOnlyNecessary] - Callback to be executed when consent with only necessary cookies
 * is detected (either given right now or already saved previously)
```

Or maybe ignore jsdoc altogether? :shrug:

Reply:
Yes, this is caused by `max-len` and I have a comment there: `ignoreComments: false, // @TODO: should we also ignore comments?`. I have no preference here but line is way too long (I must turn my editor into distraction free mode to see it whole).

Reply:
I don't have strong preference either, but this line-break mode (with space aligning from the left) just seems unpractical. But :shrug:   

However wouldn't one read the function signature mainly in the IDE? It handles the jsdoc parsing, line breaks, formatting etc. for you...

![obrazek](https://user-images.githubusercontent.com/793041/142401009-e6147303-d6c3-4a46-b3ed-1d8ed0b70415.png)

Reply:
Related - cannot we somehow inherit jsdoc from this `LmcCookieConsentManager` to `initLmcCookieConsentManager`? Because if someone uses it via `initLmcCookieConsentManager()`, there won't be any jsdoc and type hinting from the IDE at all :thinking: 

![obrazek](https://user-images.githubusercontent.com/793041/142401708-0f6a34b2-df52-4256-8582-ae1ac1deb22d.png)

Reply:
Hmm, have a look here: https://stackoverflow.com/questions/30676286/what-is-the-proper-canonical-formatting-of-long-jsdoc-lines

I will prefer the Google way. Not everyone have perfectly configured IDE and I think that the code should be readable mainly in the raw form.

Reply:
Agreed, the "Google" way with "one tab" indent looks fine, but thats not what its here. This could lead to a lot of short lines, as mentioned in the SO question.

Reply:
Now I am quite confused. Should we use **google way**
```
 * @param {function} [args.onFirstAcceptOnlyNecessary] - Callback to be executed right after only necessary cookies
 *   are accepted
```
or
**javadoc way**
```
 * @param {function} [args.onFirstAcceptOnlyNecessary] - Callback to be executed right after only necessary cookies
                                                         are accepted
```
or something different?

Reply:
"google way" - so now it looks fine, sorry, I missed you already updated the PR :)